shell/platform/darwin/ios/framework/Source/FlutterPluginAppLifeCycleDelegate.mm

```diff
@@ -4,16 +4,19 @@
 
 #import "flutter/shell/platform/darwin/ios/framework/Headers/FlutterPluginAppLifeCycleDelegate.h"
 
+#import <objc/runtime.h>
 #include "flutter/fml/logging.h"
 #include "flutter/fml/paths.h"
 #include "flutter/lib/ui/plugins/callback_cache.h"
 #import "flutter/shell/platform/darwin/ios/framework/Headers/FlutterViewController.h"
 #import "flutter/shell/platform/darwin/ios/framework/Source/FlutterCallbackCache_Internal.h"
 
 static const char* kCallbackCacheSubDir = "Library/Caches/";
+static NSString* const kApplicationDidReceiveRemoteNotificationFetchCompletionHandler =
+    @"application:didReceiveRemoteNotification:fetchCompletionHandler:";
 
 static const SEL selectorsHandledByPlugins[] = {
-    @selector(application:didReceiveRemoteNotification:fetchCompletionHandler:),
+    NSSelectorFromString(kApplicationDidReceiveRemoteNotificationFetchCompletionHandler),
     @selector(application:performFetchWithCompletionHandler:)};
```

Comment:
Does `application:performFetchWithCompletionHandler:` not also have this problem?

Reply:
I don't believe so.  It isn't listed in the issue or the pasted in message from the App Store.  The documentation for it doesn't list a need for entitlements.